en/tween/index.md

```diff
@@ -1,138 +1,19 @@
-| **delay** | Add an instant action of **delay time** |
-| **call** | Add an instant action of **call callback** |
-| **target** | Add a **instant** action to directly set the slow-motion target |
-| **union** | Package the easing action of the context into one|
-| **then** | Insert a Tween into the easing queue |
-| **repeat** | **Execution several times** (previously repeated several times, if using, please adapt) |
-| **repeatForever** | Always repeat execution |
-| **sequence** | Add a sequential slow motion |
-| **parallel** | Add a simultaneous easing |
-| **start** | Start slow motion |
-| **stop** | Stop slow motion |
-| **clone** | Clone Easing |
-| **show** | To enable rendering on the node chain, the slowing target needs to be Node |
-| **hide** | Disable rendering on the node chain, the slowing target needs to be Node |
-| **removeSelf** | Move the node out of the scene tree, the slowing target needs to be Node |
-
-### Optional attributes of to and by
-
-The definition is as follows:
-
-```typescript
-interface ITweenOption {
-    easing?: TweenEasing | ((k: number) => number);
-    progress?: (start: number, end: number, current: number, ratio: number) => number;
-    onStart?: (target: object) => {};
-    onUpdate?: (target: object, ratio: number) => {};
-    onComplete?: (target: object) => {};
-}
-```
-
-The difference with __Cocos Creator 2D__ is the addition of properties such as `onStart`, `onUpdate`, and `onComplete`. These properties are callback functions, which will be passed into the easing target when called.
-
-In addition, an additional value of the current easing will be passed in when `onUpdate` is called, and the range is `(0-1)`.
-
-#### Example of using callback
-
-Taking `onUpdate` as an example, the following code eases a position, and then setting it to multiple objects in `onUpdate`, this demonstrates __batch easing__.
-
-```typescript
-import { Node, tween, Vec3 } from 'cc';
-const nodeArray: Node[] = []; // Replace here with your node array
-const tweenTargetVec3 = new Vec3();
-tween(tweenTargetVec3)
-    .by(1, new Vec3(1, 1, 1), {
-        'onUpdate': (target: Vec3, ratio: number) => {
-            for (let i = 0; i < nodeArray.length; i++)
-                nodeArray[i].worldPosition = target;
-        }
-    });
-```
-
-## Automatic destruction
-
-In Cocos Creator 3.0, when the easing target is `Node`, it will listen to its destruction event for automatic destruction of the easing. This calls the `target` method and also automatically updates the listener.
-
-> **Note**: related test cases are located on [GitHub](https://github.com/cocos-creator/test-cases-3d).
-
-> **Note**: please refer to [Using the Tween System](https://docs.cocos.com/creator/manual/en/scripting/tween.html) documentation.
+For more details on how to use Tween, please refer to the example **Tween** [GitHub](https://github.com/cocos-creator/test-cases-3d/tree/v3.4/assets/cases/tween).
```

Comment:
Please see [GitHub](https://github.com/cocos-creator/test-cases-3d/tree/v3.4/assets/cases/tween) for sample code for using tweens.